platform/conf/conf.go

```diff
@@ -184,12 +192,7 @@ func (u *UserData) Render() (*Conf, error) {
 			plog.Warningf("rendering Container Linux config: %s", report)
 		}
 
-		// ct still returns 2.0 configs. Convert to 2.1.
-		buf, err := json.Marshal(ignc)
-		if err != nil {
-			return nil, fmt.Errorf("serializing Container Linux config: %v", err)
-		}
-		return Ignition(string(buf)).Render()
+		c.ignitionV2 = &ignc
```

Comment:
Comments indicate that previously we were converting the data into 2.1 configs, is it intentional that we're leaving them as 2.0 now?